### PR TITLE
Accessibility: Codemod from `<MdiIcon />` to `<Icon svgPath={mdiPath} aria-(hidden|label) />` (Iteration 2)

### PR DESCRIPTION
**Pull request by umpox**
**Migration automated through codemod: https://github.com/sourcegraph/codemod/pull/148**

## Description

Migrates simple `<MdiIcon />` usage directly from `mdi-react` to the `<Icon />` Wildcard component and `@mdi/js` icon content.

Changes:
- Ran codemod to automatically update all direct usage of this  icons
- Manually added `aria-label` attributes where relevant.

## Test plan

Icon edge cases tested through https://github.com/sourcegraph/sourcegraph/pull/37387, this diff is tested through automated tests.

## App preview:

- [Web](https://sg-web-tr-icon-codemod-v3-2.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-nuzrmkkrgx.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/enterprise/insights/components/views/card/InsightCard.story.tsx [hunk 1]
```diff
@@ -64,11 +63,23 @@ export const InsightCardShowcase: Story = () => (
...
+                            width="1rem"
+                        />
                     </Button>
                     <Menu>
                         <MenuButton variant="icon" className="p-1">
-                            <DotsVerticalIcon size={16} />
+                            <Icon
+                                svgPath={mdiDotsVertical}
+                                inline={false}
+                                aria-label="Options"
```
Comment (valerybugakov) [inline client/web/src/enterprise/insights/components/views/card/InsightCard.story.tsx:79]: In the diff above, `aria-label` is used on the `MenuButton` element. Is there a practical difference between these variants?

File: client/web/src/enterprise/insights/components/views/card/InsightCard.story.tsx [hunk 2]
```diff
@@ -142,11 +153,11 @@ function InsightCardWithChart() {
         <Card.Root style={{ width: '400px', height: '400px' }}>
             <Card.Header title="Insight with chart" subtitle="CSS migration insight chart">
                 <Button variant="icon" className="p-1">
-                    <FilterOutlineIcon size="1rem" />
+                    <Icon svgPath={mdiFilterOutline} inline={false} aria-hidden={true} height="1rem" width="1rem" />
                 </Button>
                 <Menu>
                     <MenuButton variant="icon" className="p-1">
-                        <DotsVerticalIcon size={16} />
+                        <Icon svgPath={mdiDotsVertical} inline={false} aria-hidden={true} height={16} width={16} />
```
Comment (valerybugakov) [inline client/web/src/enterprise/insights/components/views/card/InsightCard.story.tsx:160]: Would `aria-label` be helpful here or on the parent `MenuButton`?

File: client/web/src/enterprise/insights/components/views/card/InsightCard.story.tsx [hunk 3]
```diff
@@ -142,11 +153,11 @@ function InsightCardWithChart() {
         <Card.Root style={{ width: '400px', height: '400px' }}>
             <Card.Header title="Insight with chart" subtitle="CSS migration insight chart">
                 <Button variant="icon" className="p-1">
-                    <FilterOutlineIcon size="1rem" />
+                    <Icon svgPath={mdiFilterOutline} inline={false} aria-hidden={true} height="1rem" width="1rem" />
```
Comment (valerybugakov) [inline client/web/src/enterprise/insights/components/views/card/InsightCard.story.tsx:156]: Would `aria-label` be helpful here or on the parent `Button`?

File: client/web/src/org/members/AddMemberToOrgModal.tsx [hunk 4]
```diff
@@ -78,7 +78,7 @@ export const AddMemberToOrgModal: React.FunctionComponent<
                         <H3>{title}</H3>
                         <Button className={classNames('btn-icon', styles.closeButton)} onClick={onCloseAddUserModal}>
                             <VisuallyHidden>Close</VisuallyHidden>
-                            <CloseIcon />
+                            <Icon svgPath={mdiClose} inline={false} aria-hidden={true} />
```
Comment (valerybugakov) [inline client/web/src/org/members/AddMemberToOrgModal.tsx:81]: Similar question [to the one in the previous PR](https://github.com/sourcegraph/sourcegraph/pull/38717/files#r920652458) for visibility.

**Review by valerybugakov — APPROVED** [review]
The migration looks excellent!
Left a couple of questions related to `aria-label` attributes.
